README.md

```diff
@@ -38,6 +38,9 @@ Version](https://img.shields.io/github/r-package/v/insightsengineering/rlistings
 Issues](https://img.shields.io/github/issues-raw/insightsengineering/rlistings?color=red&label=open%20issues)](https://github.com/insightsengineering/rlistings/issues?q=is%3Aissue+is%3Aopen+sort%3Aupdated-desc)
 <!-- end badges -->
 
+<img src="man/figures/rlistings-logo.svg" align="right" alt="" width="180">
```

Comment:
I think it is better to put it on the right of the title as it is in `teal`:

`# Teal: Interactive Exploratory Data Analysis with Shiny Web-Applications <a href='https://insightsengineering.github.io/teal/'><img src="man/figures/teal.png" align="right" height="139" style="max-width: 100%;"/></a  >`

Reply:
what do you think @shajoezhu ?

Reply:
I was more thinking of similar to rtables actually

see 

```markdown
# rtables <a href='https://github.com/Roche/rtables'><img src="man/figures/rtables_logo_small.png" align="right" /></a>
```

https://raw.githubusercontent.com/Roche/rtables/main/README.md

Reply:
what do you reckon? @Melkiades @gmbecker

Reply:
it is fine both imo